tests/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+set(TESTING_TIMEOUT 2000)
```

Comment:
It was not debug version on Windows, I guess :)
Flat-nodes test can take even more at 20-30 mb/s 20Gb generating several times. I suggest to make it a parameter, 1200 by default (to be sure Travis will not hang).